### PR TITLE
Added about modal design docs

### PR DESCRIPTION
**Pull request by stacymcauliffe**
I added design docs for the about modal component

### CODE REVIEW

File: patternfly-4/components/about-modal/design.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# About Modal
+The About Modal stores the version number(s) of the application as well as any appropriate legal text.
+
+## Elements of an About Modal
+![Modal](img/about-modal.png)
+1. Backdrop
+2. Modal box
+3. Graphics
```
Comment (mcarrano) [inline patternfly-4/components/about-modal/design.md]: I wonder if we need to say more about this. Seems like this area is being used for a brand logotype. Something like "substitute your brand here"?

File: patternfly-4/components/about-modal/design.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# About Modal
+The About Modal stores the version number(s) of the application as well as any appropriate legal text.
+
+## Elements of an About Modal
+![Modal](img/about-modal.png)
```
Comment (mcarrano) [inline patternfly-4/components/about-modal/design.md]: We should try to get these images a bit larger so they are more readable.

File: patternfly-4/components/about-modal/design.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  * When version and build information are both shown: Version 6.3 (Build 5)
+  * When version only: Version 6.3
+* When build only: Build 5
+7. Close
+8. Background image
+
+## Usage
+Use the About Modal to provide information about the product including the product name, version numbers, and any appropriate legal text.
+
+## Variations
```
Comment (mcarrano) [inline patternfly-4/components/about-modal/design.md]: Not sure I understand what these variations are for. When would I use them?

**Comment by stacymcauliffe** [comment]
@mcarrano -- made updates. Let me know if this works for you. 

Thanks

**Comment by stacymcauliffe** [comment]
@mcarrano - added changes based on our convo with kyle and michael. (focus on the latest commit. I jumped the gun twice.)

**Pull request merged**